junit/src/main/java/cucumber/api/junit/Cucumber.java

```diff
@@ -28,7 +31,8 @@
 /**
  * <p>
  * Classes annotated with {@code @RunWith(Cucumber.class)} will run a Cucumber Feature.
- * The class should be empty without any fields or methods.
+ * In general, the runner class should be empty without any fields or methods.
+ * However, it is possible to use the following JUnit annotations in the runner class: {@link ClassRule}, {@link BeforeClass} and {@link AfterClass}.
```

Comment:
I think this should be rephrased to: "Cucumber supports the {@link ClassRule}, {@link BeforeClass} and {@link AfterClass} annotations." and moved to the end of the java doc.


In this instance I think it should be "should not"